framework/source/class/qx/theme/manager/Font.js

```diff
@@ -224,11 +224,11 @@ qx.Class.define("qx.theme.manager.Font",
 
           // Inject information about custom charcter set tests before we apply the
           // complete blob in one.
-          if (source[key].testCharacters) {
-            fo.setTestCharacters(source[key].testCharacters);
+          if (source[key].comparisonString) {
+            fo.setComparisonString(source[key].comparisonString);
           }
 
-          dest[key] = (new font).set(source[key]);
+          dest[key] = fo.set(source[key]);
```

Comment:
no :) because the code never worked ... afaik you can not set 'testCharacters' from the theme ... since that property does not exist. The property in qooxdoo is called 'comparisonString' ... see the documentation: http://www.qooxdoo.org/5.1/pages/development/icon_fonts.html

even the compiler knows ... 

https://github.com/qooxdoo/qooxdoo-compiler/blob/b4d5f4c1bdeabc38846bb79a3c2ef7dc3387cd45/lib/qx/tool/compiler/app/WebFont.js#L224

why, the property is called `testCharacters` in the Manifest.json ... I don't know ... and I would sugest to change that too ... I am pretty sure noone would miss it:

https://github.com/qooxdoo/qooxdoo-compiler/search?utf8=%E2%9C%93&q=testCharacters&type=Code

since it is not even documented :)